http-server/pom.xml

```diff
@@ -220,4 +232,17 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <!-- Following can be removed whenever we update airlift/airbase to recent version -->
+    <build>
```

Comment:
What exactly is the conflict here? Can we resolve the warning and upgrade some things?

I don't get exactly what this is doing that avoids a warning.

Reply:
We are using a dated version of `org.basepom.maven.duplicate-finder-maven-plugin` which comes from facebook's airlift/airbase. Because of this, we are getting `duplicate and different classes warning` for `META-INF.versions.9.module-info`  which is causing maven build failure. This is fixed in the `org.basepom.maven.duplicate-finder-maven-plugin` recent versions, that's why I am getting a recent version of this dependency for building http-server and this can be removed once we update the duplicate finder in facebook's airbase. Didn't want to update it now as the change will be in a different repo and might come up with other changes which might take up a lot of time.
The exact warning is as follows:
```
[WARNING] Found duplicate and different classes in [org.bouncycastle:bcpkix-jdk15on:1.68, org.bouncycastle:bcprov-jdk15on:1.68]:
[WARNING]   META-INF.versions.9.module-info
[WARNING] Found duplicate classes/resources in test classpath.
```